scripts/apps/ingest/directives/IngestSourcesContent.js

```diff
@@ -79,6 +79,18 @@ export function IngestSourcesContent(ingestSources, gettext, notify, api, $locat
                     13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24];
                 $scope.ingestExpiry = deployConfig.getSync('ingest_expiry_minutes');
 
+                $scope.statusFilters = [
+                    {label: gettext('Active'), id: 'active'},
+                    {label: gettext('All'), id: 'all'},
+                    {label: gettext('Closed'), id: 'closed'},
+                ];
+
+                $scope.activeStatusFilter = $scope.statusFilters[0];
+                $scope.filterIngestSources = function(id) {
+                    $scope.activeStatusFilter = _.find($scope.statusFilters, (item) => item.id === id);
```

Comment:
Use [`Array.prototype.find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) instead of lodash